client/src/Components/Tasks/TaskListItem.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { FlexRow, FlexColumn, Divider } from "custom-components";
+import moment from "moment";
+import { Checkbox, Label } from 'semantic-ui-react';
+import Tasks from "./Tasks";
+
+const TaskListItem = ({ task }) => {
+  return (
+    <FlexRow alignCenter justifyBetween width="full" style={{marginTop: "5px"}}>
+      <FlexColumn>
+        <Checkbox label={task.description} />
+        <p style={{ paddingTop: "10px", paddingLeft: "25px" }}>{task.assignedTo.firstName} {task.assignedTo.lastName} - {moment(task.endDate).format("MM/DD")}</p>
```

Comment:
This line is trying to display task.assigned.firstName and task.assigned.lastName, but "assigned" is not part of the task state.

Reply:
@mglittleton can you explain what you mean by 'task state'?

Reply:
@mglittleton I just took out those fields for now. Does that still return errors for you? Once I get this merged into development I can use it for reference if I try displaying tasks another way...

Reply:
@LBWright , I just meant the part of the store under "tasks". It has an array of tasks with most of the info (property, start and end dates, etc), but is missing the assignedTo field for some reason.

Reply:
@BryceEvans it's rendering everything fine now without the names. Question though, wouldn't you only want to see the tasks that are assigned to the user? There should be no reason for that to be part of the task list itself.